frontend/src/components/pages/AnalyticsPage.tsx

```
@@ -115,13 +110,13 @@ const pieData = [
-  repoData: any;
+    repoData: RepositoryData | null;
 }
 
 const AnalyticsPage: React.FC<Props> = ({ repoData }) => {
-  if (!repoData || !repoData.pull_requests) {
-    return <div>No data available. Please analyze a repository first.</div>;
-  }
+    if (!repoData || !repoData.pull_requests) {
+        return <div>No data available. Please analyze a repository first.</div>;
+    }
     const [selectedRange, setSelectedRange] = React.useState('Last Week');
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)